README.md

```diff
@@ -75,16 +75,67 @@ The µDNS library is published under the 2 clause BSD license.
 You first need to install [OCaml](https://ocaml.org) (at least 4.04.0) and
 [opam](https://opam.ocaml.org), the OCaml package manager (at least 2.0.0) on
 your machine (you can use opam to install an up-to-date OCaml (`opam switch
-4.07.1`)).  You may want to follow the [mirage installation
+4.07.1`)).
+
+You may want to follow the [mirage installation
 instructions](https://mirage.io/wiki/install) to get `mirage` installed on your
 computer.
 
-µDNS is not released yet, but you can install it and its dependencies via opam:
-`opam pin add dns https://github.com/roburio/udns.git`
+To lower the amount of run-time dependencies for each individual functionality,
+the library is split across a number of opam packages.
+
+µDNS is not released yet, but you can install it and its dependencies via opam, see [Development](#Development).
```

Comment:
this used to be 80 characters wide, now it is no longer... would you mind inserting a linebreak after the `,`?